docs/reference/release/release_0.0.2.rst

```diff
@@ -34,7 +34,7 @@ API Changes
 -----------
 - ``check_lcc`` argument in ``AdjacencySpectralEmbed``, ``LaplacianSpectralEmbed``, and ``OmnibusEmbed`` classes, which checks if input graph(s) are fully connected when ``check_lcc`` is True.
 - ``gridplot`` and ``heatmap`` now have a ``inner_hier_labels`` and ``outer_hier_labels``, which are used for hierarchical labeling of nodes.
-- ``to_laplace`` function now has ``regularizer`` arg for when ``form`` is 'R-DAD'.
+- ``to_laplacian`` function now has ``regularizer`` arg for when ``form`` is 'R-DAD'.
```

Comment:
this PR shouldn't have to update all this stuff

Reply:
lemme create a new pr